File: poo/Cpp/IMC/IMC.cpp
#include <iostream>
#include <string>

int main() {
    double altura, peso;
    std::cin >> altura >> peso;
    
    double imc = peso / (altura * altura);

    if (imc < 16) {
        std::cout << "Magreza grave" << std::endl;
    } else if (imc < 17) {
        std::cout << "Magreza moderada" << std::endl;
    } else if (imc < 18.5) {
        std::cout << "Magreza leve" << std::endl;
    } else if (imc < 25) {
        std::cout << "Saudavel" << std::endl;
    } else if (imc < 30) {
        std::cout << "Sobrepeso" << std::endl;
    } else if (imc < 35) {
        std::cout << "Obesidade Grau I" << std::endl;
    } else if (imc < 40) {
        std::cout << "Obesidade Grau II (severa)" << std::endl;
    } else {
        std::cout << "Obesidade Grau III (morbida)" << std::endl;
    }

    return 0;
}

File: poo/Java/IMC/IMC.java
import java.util.Scanner;

public class IMC {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double altura = scanner.nextDouble();
        double peso = scanner.nextDouble();
        double imc = peso / (altura * altura);

        if (imc < 16) {
            System.out.println("Magreza grave");
        } else if (imc < 17) {
            System.out.println("Magreza moderada");
        } else if (imc < 18.5) {
            System.out.println("Magreza leve");
        } else if (imc < 25) {
            System.out.println("Saudavel");
        } else if (imc < 30) {
            System.out.println("Sobrepeso");
        } else if (imc < 35) {
            System.out.println("Obesidade Grau I");
        } else if (imc < 40) {
            System.out.println("Obesidade Grau II (severa)");
        } else {
            System.out.println("Obesidade Grau III (morbida)");
        }
        scanner.close();
    }
}